### DISCUSSION TITLE
Proper number of cells pass filter

### DISCUSSION THREAD

**Discussion by tao-bioinfo** [GreenleafLab/ArchR#909] [Questions / Documentation] [answered]
In https://www.archrproject.com/bookdown/per-cell-quality-control.html it says,

```
By default in ArchR, pass-filter cells are identified as those cells having a TSS enrichment score greater than 4 and more than 1000 unique nuclear fragments. It is important to note that the actual numeric value of the TSS enrichment score depends on the set of TSSs used. The default values in ArchR were designed for human data and it may be important to change the default thresholds when running createArrowFiles().
```

Obviously, the lower thresholds we set, the more cells would pass filter. My problem is: **which number of cells is proper to do all the downstram analysis?**

For example, I used a thredshold: minTSS=2.5, and the number of passed cells for six samples were:

sample1:	1674
sample2:	314
sample3:	1655
sample4:	763
sample5:	298
sample6:	2618

**Comment by rcorces** [comment, marked as answer]
> which number of cells is proper to do all the downstram analysis?

This is not the right way to think about this problem. You dont want to include cells with bad quality metrics just to obtain a sufficient number of cells. If you dont have enough pass filter cells to analyze, you need to generate more data. There is no magic number for the right number of cells but I feel like any cluster with fewer than 100-200 cells is not going to generate data of sufficient quality to analyze.
